django/contrib/staticfiles/storage.py

```diff
@@ -403,7 +403,7 @@ def load_manifest(self):
 
     def post_process(self, *args, **kwargs):
         self.hashed_files = OrderedDict()
-        all_post_processed = super(ManifestFilesMixin,
+        all_post_processed = super(,
```

Comment:
It seems you'll need to check the patch by hand. This is an obvious syntax error.